### ISSUE TITLE
Java package for 0.3.4 not available in public registries

### ISSUE THREAD

**Issue by davechallis** [duckdb/duckdb#3683]
The installation instructions for Java on the website say to use: 
```
<dependency>
 <groupId>org.duckdb</groupId>
 <artifactId>duckdb_jdbc</artifactId>
 <version>0.3.4</version>
</dependency>
```
However `0.3.4` doesn't exist in most of the package repositories I've looked at, e.g. https://search.maven.org/search?q=duckdb shows `0.3.3` as the latest release.

**Comment by Mytherin** [comment]
0.3.4 and 0.3.3 are functionally almost identical, the reason we released 0.3.4 was to fix issues with missing Python builds. Indeed we should fix the website here.

**Comment by Mytherin** [comment]
This should be fixed on the website now
